CalendarPicker/Day.js

```diff
@@ -41,12 +41,21 @@ export default function Day(props) {
     disabledDatesTextStyle,
     minRangeDuration,
     maxRangeDuration,
-    enableDateChange
+    enableDateChange,
+    renderDay
   } = props;
 
   const thisDay = new Date(year, month, day, 12);
   const today = new Date();
 
+  const DayComponent = ({ style }) => {
```

Comment:
it'd be worth memoizing this to reduce re-render penalties

Reply:
Good idea